### DISCUSSION TITLE
music/chroma_console/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#1] [General]
# music/chroma_console/

https://shanenull.com/music/chroma_console/

**Comment by shane0** [comment]
https://www.elektronauts.com/t/chroma-console-hologram-electronics/202802/368

**Comment by shane0** [comment]
https://youtu.be/sqF-s3TW3Uw

**Comment by shane0** [comment]
https://www.hologramelectronics.com/chroma-console

**Comment by shane0** [comment]
![m](https://private-user-images.githubusercontent.com/1454458/294726406-5ff1c744-7030-4534-b3b0-bb715fe384a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ1OTIyMTEsIm5iZiI6MTcwNDU5MTkxMSwicGF0aCI6Ii8xNDU0NDU4LzI5NDcyNjQwNi01ZmYxYzc0NC03MDMwLTQ1MzQtYjNiMC1iYjcxNWZlMzg0YTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMDdUMDE0NTExWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjcyNDY1MWQwNGIyZjFjMGNiZjg0M2JlZTQ0NmJhZDI5OTI2ZDA1YzllNDFjNDdlNGY2YzA2OGFhZjkyMTBiYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.eOfAepXKlwzo2GKLmYFhYYmUxZ83fJOe6zjaJSIY3l0)

**Comment by shane0** [comment]
<img width="403" alt="image" src="https://github.com/shane0/music/assets/1454458/2fad0207-a62a-4cbc-858b-f295ab427be4">
